### ISSUE TITLE
What is x1,y1,x2,y2 in sample_parser.py?

### ISSUE THREAD

**Issue by ManjeeraJagiri** [yhenon/keras-frcnn#168]
Is x1=xmin,y1=ymin,x2=xmax,y2=ymax as given in pascal voc format? Is it possible to train on own images using pascal voc format for the annotations rather than the .txt file?

**Comment by Abhijit-2592** [comment]
yes, you are right about x1,y1,x2,y2. To put it simply those are the coordinates of the leading diagonal of the bbox.
And it is possible to train own images using pascal voc format for annotations . You must modify pascal_voc_parser accordingly. But I don't see why you would do that. Using the text file is easier compared to that

**Comment by ManjeeraJagiri** [comment]
@Abhijit-2592 , Thanks for your reply!
